db_schema.puml

```diff
@@ -0,0 +1,133 @@
+@startuml
```

Comment:
why we need this?

Reply:
This is a structure annotation for **.puml** files to know the plugin that reads these files that this is the start of the specific file